src/components/BiobankExplorerContainer.vue

```diff
@@ -27,12 +27,12 @@
 </style>
 
 <script>
-  import BiobankCardsContainer from './cards/BiobankCardsContainer'
```

Comment:
Do we also want this for `./` paths? I kind of like that the module keeps working if you move it around.

Reply:
yes we want it , thats the whole idea, that it keeps working if you move it around